### PR TITLE
Migrate away from CRA

### LINKED ISSUE #1001: Replace deprecated blueprintjs components
**Is your feature request related to a problem? Please describe.**
We use a few deprecated blueprintjs components. This could lead to bugs unnoticed.

**Describe the solution you'd like**
Replace all deprecated components with the 'newer' counterparts.

Deprecated Components:
- (DateInput)
- IResizeEntry
- Select

**Describe alternatives you've considered**
/

**Additional context**
The deprecated blueprintjs date input was removed, because it didn't work properly. The newer component might fix this (we currently use dateinput from material, as it just did not work properly). #1003 

Updating date input is blocked by https://github.com/palantir/blueprint/issues/5699

### PR DESCRIPTION
**Pull request by michael-markl**
Since create-react-app is mostly unmaintained, we should likely move away from it soon.
Here's a suggestion of a first move towards that goal: I used the `eject` command and cleaned up a bit the configs (and moved them over to TS). Of course, they are still quite "verbose" and might be partially outdated, but at least filled with a lot of helpful commands.
Also, there are quite some dev-dependencies to the remaining CRA-stack (like "react-dev-utils" or "babel-preset-react-app"). We should probably move away from them, too.

Nevertheless, this upgrades all npm dependencies to recent versions, removing many warnings and deprecated packages. (This reduces the `package-lock.json` file by ~30%)

Also closes #1001 .

### CODE REVIEW

**Review by sarahsporck — APPROVED** [review]
Tested in firefox. Everything still seems to work nicely. Thx for the commit. So the next stage would be to get rid of the babel stuff. Would you mind creating a ticket for this?

File: administration/package.json [hunk 1]
```diff
@@ -2,69 +2,101 @@
...
-    "@blueprintjs/popover2": "^1.13.3",
-    "@blueprintjs/select": "^4.9.3",
-    "@blueprintjs/table": "^4.9.3",
-    "@bufbuild/protobuf": "^1.0.0",
-    "@fontsource/roboto": "^4.5.8",
-    "@mui/icons-material": "^5.11.9",
-    "@mui/material": "^5.11.10",
-    "@zxing/library": "^0.19.2",
+    "@apollo/client": "^3.7.16",
+    "@blueprintjs/core": "^5.0.0",
```
Comment (michael-markl) [inline administration/package.json:10]: [Also, if I remember correctly, removing the dependency will result in a compilation error.]

File: administration/package.json [hunk 2]
```diff
@@ -2,69 +2,101 @@
...
+    "@blueprintjs/popover2": "^2.0.0",
+    "@blueprintjs/select": "^5.0.0",
+    "@blueprintjs/table": "^5.0.0",
+    "@bufbuild/protobuf": "^1.2.1",
+    "@emotion/styled": "^11.11.0",
```
[the remark this answers is not in the document]
Comment (sarahsporck) [reply administration/package.json:15]: Ahh okay. I was wondering why this was there. I think for now it is not important that we use both emotion and styled components @f1sh1918 ?

[the remark this answers is not in the document]
Comment (f1sh1918) [reply administration/package.json:15]: i'm fine with that

**Review by f1sh1918 — APPROVED** [review]
sorry for letting you wait @michael-markl 
looks fine to me. roughly tested druckerei functions

**Pull request merged**